### PR TITLE
Remove _color-pallette.scss and replace those vars with PatternFly eq…

### PR DESCRIPTION
**Pull request by rhamilto**
…uivalents and remove orphaned `.ReactModal__Content*` rules.

The most obvious of these changes is the background color in `.well`.  @openshift/team-ux-review are y'all good with the new bg color? 

Before:
console apps ui-preserve origin-gce dev openshift com_k8s_ns_robb_serviceinstances_mongodb-persistent

After:
localhost_9000_k8s_ns_robb_serviceinstances_mongodb-persistent

### CODE REVIEW

**Comment by rhamilto** [comment]
/hold 

for feedback from ux.

**Comment by rhamilto** [comment]
/cancel hold

Ready for review

**Comment by cshinn** [comment]
@rhamilto Looks good to me

File: frontend/public/style/_common.scss [hunk 1]
```diff
@@ -121,7 +121,7 @@
 }
 
 .cos-error-title {
-  color: $red;
+  color: $color-pf-red;
```
Comment (sg00dwin) [inline frontend/public/style/_common.scss]: Duplicate rule also in `_status-box.scss`

Comment (rhamilto) [reply frontend/public/style/_common.scss]: Both are needed.  This one is the generic one.  The `_status-box.scss` one is needed to override the color change 6 lines up.

File: frontend/public/style/_status.scss [hunk 2]
```diff
@@ -7,21 +7,21 @@
 }
 
 .co-m-status--critical {
-  color: $red-dark;
+  color: $color-pf-red-200 ;
```
Comment (sg00dwin) [inline frontend/public/style/_status.scss]: `$red-invalid` and `$red-dark` are both(?) being changed to `$color-pf-red-200`
If `$red` becomes `$color-pf-red` which is `$color-pf-red-300`, then shouldn't `$red-dark` become `$color-pf-red-400`

**Comment by spadgett** [comment]
jenkins rebuild

**Comment by beanh66** [comment]
@rhamilto What is the color you are using for the new blue background? And what is the font color? I wanted to loop @rawsyntax in so he can reuse the component for another use case.

File: frontend/public/style/_common.scss [hunk 3]
```diff
@@ -234,8 +234,8 @@
 }
 
 .co-well {
-  color: $blue-darker;
-  background-color: $blue-bg;
+  color: $color-pf-blue-600;
```
Comment (rhamilto) [inline frontend/public/style/_common.scss]: @beanh66, @rawsyntax well text color

File: frontend/public/style/_common.scss [hunk 4]
```diff
@@ -234,8 +234,8 @@
...
+  background-color: $color-pf-blue-50;
```
Comment (rhamilto) [inline frontend/public/style/_common.scss]: @beanh66, @rawsyntax well bg color

**Pull request merged**